api/ring-camera.ts

```diff
@@ -245,10 +245,16 @@ export class RingCamera {
     }
 
     await this.restClient.request({
-      method: 'PUT',
+      method: 'put',
       url: this.doorbotUrl(),
-      data: {
-        'doorbot[settings][chime_settings][enable]': on,
+      form: {
+        doorbot: {
+        	settings: {
+        		chime_settings: {
+        			enable: 'on',
+        		},
+        	},
+        },
```

Comment:
Nice! Way better than what I had before

Reply:
Actually there is a small bug in here. The 'on' argument of the function is not respected, as I wrongly typed 'on' instead of on. 🤦